book/website/ethical-research/ethics-case-study.md

```diff
@@ -0,0 +1,134 @@
+# Research Ethics Committees Processes
+
+Research ethics committees (RECs), also known as institutional review boards (IRBs), exist within institutions to scrutinise the ethical implications of their research projects.
+RECs have historically focused on projects that involve human participants, meaning that they often prioritis the well-being of those involved in research projects.
+RECs are also a useful mechanism for the mitigation of reputational damage for their institutions, as they can ensure that research is conducted within the law.
+RECs are also well-positioned to improve the state of scientific inquiry.
```

Comment:
These two sentences both begin "RECs are also", it's a bit clumsy to read.

Reply:
Sorted!